pgppacket.c

```diff
@@ -119,7 +119,8 @@ unsigned char *pgp_read_packet (FILE * fp, size_t * len)
 	  perror ("fread");
 	  goto bail;
 	}
-	material += b + 192;
+	material += b;
+	material += 192;
```

Comment:
`b` is un unsigned char which is read from outside of mutt.
If `b` is large, then `b + 192` will overflow before it's added to `material`.

An alternative solution would be to cast `b` to the type of `material` first, then add.